models/balancer/gnosis/balancer_v2_gnosis_pools_tokens_weights.sql

```diff
@@ -0,0 +1,33 @@
+{{
+    config(
+        schema='balancer_v2_gnosis',
+        alias='pools_tokens_weights',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['pool_id', 'token_address'],
+        post_hook='{{ expose_spells(\'["gnosis"]\',
+                                    "project",
+                                    "balancer_v2",
+                                    \'["metacrypto", "jacektrocinski", "viniabussafi"]\') }}'
+    )
+}}
+
+--
+-- Balancer v2 Pools Tokens Weights
+--
+
+SELECT
+    registered.poolId AS pool_id,
+    tokens.token_address,
+    weights.normalized_weight / POWER(10, 18) AS normalized_weight
+FROM {{ source('balancer_v2_gnosis', 'Vault_evt_PoolRegistered') }} registered
+INNER JOIN {{ source('balancer_v2_gnosis', 'WeightedPoolV2Factory_call_create') }} call_create
```

Comment:
we're currently at WeightedV4 - have we sent new factories for decoding?

Reply:
just sent.